libraries-bom/pom.xml

```diff
@@ -52,15 +52,15 @@
     <google.cloud.core.version>2.7.1</google.cloud.core.version>
     <io.grpc.version>1.46.0</io.grpc.version>
     <http.version>1.41.8</http.version>
-    <protobuf.version>3.21.0</protobuf.version>
+    <protobuf.version>3.20.1</protobuf.version>
     <!-- We don't use gax-bom because it includes the artifacts with 'testlib' classifier.
         When updating gax.version, update gax.httpjson.version too. -->
     <gax.version>2.18.1</gax.version>
     <gax.httpjson.version>0.103.1</gax.httpjson.version>
     <auth.version>1.7.0</auth.version>
     <api-common.version>2.2.0</api-common.version>
     <common.protos.version>2.8.3</common.protos.version>
-    <iam.protos.version>1.4.0</iam.protos.version>
+    <iam.protos.version>1.3.4</iam.protos.version>
```

Comment:
Memo: There's a planned enhancement for phase 2 where the Libraries BOM use the first-party-dependencies BOM (a part of the shared dependencies BOM). With that, we don't have to manually maintain the individual versions of the libraries.